package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.0",
   "scripts": {
     "dev": "vite serve src",
-    "build": "vite build",
-    "serve": "vite preview"
+    "build": "vite build src",
+    "serve": "vite preview src"
```

Comment:
This is to create a production build from the correct entry point, the src directory